docs: update the FAQ about `Instant Finality` (#194)
godwokenrises/godwoken-doc: docs/faq.md

----

### Instant Finality: Getting the transaction completed faster

> Q: The average block confirmation interval on Godwoken is defined by [block_interval_secs](https://github.com/godwokenrises/godwoken/blob/v1.14.0/crates/config/src/config.rs#L184-L185), defaut to 8s. If I would like to have my transactions completed more quickly, what should I do to make this happen?

Author: ```suggestion
Q: The average block confirmation interval on Godwoken is defined by [block_interval_secs](https://github.com/godwokenrises/godwoken/blob/v1.14.0/crates/config/src/config.rs#L184-L185), which is set to 8s by default. If I want to have my transactions completed more quickly, what should I do to make this happen?
```
